projects/angular-cesium/src/lib/angular-cesium-widgets/models/polyline-editor-observable.ts

```diff
@@ -8,4 +8,5 @@ import { Cartesian3 } from '../../angular-cesium';
 export class PolylineEditorObservable extends EditorObservable<PolylineEditUpdate> {
   setManually: (points: { position: Cartesian3, pointProp?: PointProps }[] | Cartesian3[], polylineProps?: PolylineProps) => void;
   getCurrentPoints: () => EditPoint[];
+  finishCreation: () => boolean;
```

Comment:
I thought about implementing the same logic to all editors, but only polyline and polygon are editors that finish with a special event from the user that signals the end of the drawing, the others don't:
1. Point editor - on tap on the map and the drawing ends, so no need for a "end drawing" button
2. hippodrome/circle/ellipse/rectangle - one tap starts the drawing, another one finishes it. No need for a "end drawing" button because after the first click the user haven't finished the drawing, and after the seconds one the drawing is automatically done.
3. Polygon/Polyline - need the "finish drawing" button to replace the "addLastPointEvent"

Reply:
Mmm I am not sure about that
I agree if the use case is a button then the hippodrome/circle/ellipse/rectangle finishCreation() is redundant,
but there could be a variety of other use cases when you would like to programmatically finish drawing. For example, stop drawing automatically after a certain amount of time.
So if it is not too big hassle i think implementing it for all editors and making the API consist would be the best.

Reply:
No problem, done 👍🏻